apps/maestro/src/features/RegisterRemoteTokens/RegisterRemoteTokens.tsx

```diff
@@ -94,33 +100,35 @@ export const RegisterRemoteTokens: FC<RegisterRemoteTokensProps> = (props) => {
     await recordRemoteTokenDeployment({
       tokenAddress: props.tokenAddress,
       chainId: props.originChainId ?? -1,
-      deploymentMessageId: `${digest}-${txIndex}`,
+      // TODO: find event Txindex correctly
+      deploymentMessageId: `${digest}`,
```

Comment:
the event index needed to be appended here as well

Reply:
there's is no transaction index in sui, should we hardcode a 0?

Reply:
there's event index. for example, this tx https://devnet-amplifier.axelarscan.io/gmp/AruX9g1GWvSQmF1ZzZuW1uaTEi3fiJXgkWMEFXmKF4vU-1, In this url, you can notice that the format is `digest-eventIndex`. The eventIndex `1` is from the order of `ContractCall` event in this [sent tx](https://suiscan.xyz/testnet/tx/AruX9g1GWvSQmF1ZzZuW1uaTEi3fiJXgkWMEFXmKF4vU) (scroll down to the buttom)
![image](https://github.com/user-attachments/assets/02697790-64ef-466c-b4a4-46b463584d3c)